templates/telco/index.html

```diff
@@ -278,371 +208,374 @@ <h2 class="p-muted-heading u-no-max-width">OPEN SOURCE COMES THROUGH CONTRIBUTIO
+        Benefit from full automation, maximum security and a reduced footprint optimised for the edge hardware.
       </p>
-      <p>
-        <a class="p-button" href="https://jaas.ai/openstack">
-          <span class="p-link--external">
-            Learn more about Charmed OpenStack
-          </span>
-        </a>
+      <p class="u-sv2">
+        In response to increasing demands for high bandwidth and low latency, Canonical provides comprehensive solutions for both near and far edge. From micro clouds to devices.
       </p>
       <p>
-        <a href="/engage/charmed-openstack-adoption-whitepaper">Read the whitepaper - “A guide to a successful OpenStack adoption and deployment”&nbsp;&rsaquo;</a>
+        <a  href="/engage/edge-month">Watch a series of webinars about Canonical's solutions for the edge&nbsp;&rsaquo;</a>
       </p>
     </div>
-    <div class="col-4 u-align--center u-hide--small">
+    <div class="col-5 u-align--center u-vertically-center u-hide--small">
       {{
         image(
-        url="https://assets.ubuntu.com/v1/95212c08-OpenStack_Logo_2016.svg",
+        url="https://assets.ubuntu.com/v1/a84416b2-IoT-gateways-on-ubuntu.svg",
         alt="",
-        height="106",
-        width="220",
+        height="400",
+        width="680",
         hi_def=True,
-        loading="lazy"
+        loading="auto"
         ) | safe
       }}
     </div>
   </div>
-  <div class="u-fixed-width">
-    <hr class="p-separator">
-  </div>
-  <div class="row u-vertically-center">
-    <div class="col-4 u-align--center u-hide--small">
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-10">
+      <h2 class="u-sv3">The entire NFV stack</h2>
+      <p>
+        NFVI and MANO are based on established open source technologies, designed for hosting mission-critical VNFs and CNFs with the most demanding performance and security requirements.
+      </p>
+      <div class="u-align--center u-hide--small u-sv4">
       {{
         image(
-        url="https://assets.ubuntu.com/v1/767f38a4-kubernetes-stacked-color.svg",
+        url="https://assets.ubuntu.com/v1/adca64ed-Untitled.png",
         alt="",
-        height="156",
-        width="200",
+        height="450",
+        width="800",
         hi_def=True,
-        loading="lazy"
+        loading="auto"
         ) | safe
       }}
+      </div>
+      <p>
+        <a href="/openstack/contact-us" class="p-button--positive">Download datasheet</a>
```

Comment:
This link won't lead to a datasheet

Reply:
Wups, I removed the link as in the copy doc it links to another copy doc of a page which hasn't been built yet (correct me if I'm wrong?)